.github/workflows/promote-nightly.yml

```diff
@@ -0,0 +1,25 @@
+# Promotes a new build to the Nightly channel on a schedule.
+
+name: Promote Nightly Channel
+
+on:
+  repository_dispatch:
+    types: 'promote-nightly'
+    inputs:
+      amp-version:
+        description: 'AMP version number (13 digits)'
+        required: true
+
+  workflow_dispatch:
+    inputs:
+      amp-version:
+        description: 'AMP version number (13 digits)'
+        required: true
+
+jobs:
+  promote-nightly:
+    uses: ampproject/cdn-configuration/.github/workflows/promote-reusable-workflow.yml@main
+    with:
```

Comment:
Have you tried running this on your own fork? I don't see `jobs.<job_id>.with` as a valid option on https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions

Reply:
I haven't tried running it locally, figured we can try it after we merge :P
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idwith

Reply:
Thanks for the link. Can you test locally? Would help with so much of our back-and-forths today...

Reply:
I can try, see how involved it might be to test it on a separate repo - if it's gonna take >1h I prefer to just merge this PR and fix anything that goes wrong here

Reply:
(will keep you posted)

Reply:
Hopefully not too involved. I do it myself for the workflows I write. Reviewing this PR has taken a lot of my own time today. Thank you!

Reply:
Tested this on my own fork as you suggested and I'm so glad that you did, because it broke in all the places I didn't expect it to break! 5138769 fixes everything that I learned from the fork tests:
* Logs for the fork's successful workflow run: https://github.com/danielrozenberg/cdn-configuration/runs/4732259272?check_suite_focus=true#step:5:1
* PR that was auto generated: https://github.com/danielrozenberg/cdn-configuration/pull/1